index.html

```diff
@@ -1,3 +1,29 @@
-<html>
-  <h1>hello main</h1>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <h1>
+            My Logo
+        </h1>
+        <a href="#">Portfolio</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
```

Comment:
- [ ] For the mobile version we don't these menus at this moment, let us use the hamburger button as suggested.

The hamburger button is on the right, and has a nav tag

Reply:
please clarify further. the hamburger should come when the web is resized or. Your help is much appreciated

Reply:
Yeah it should come when the web page is resized, you can keep that in our code, however hide it when you resize the page.